engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/CreateGameScreen.java

```diff
@@ -138,10 +139,20 @@ public void set(Module value) {
         gameplay.setOptionRenderer(new StringTextRenderer<Module>() {
             @Override
             public String getString(Module value) {
-                return value.getMetadata().getDisplayName().value();
+            	return value.getMetadata().getDisplayName().value();
+            }
+            
+            @Override
+            public void draw(Module value, Canvas canvas)
+            {
+        		canvas.getCurrentStyle().setTextColor(
```

Comment:
Again, tab.

Change the default indentation mode in your IDE/text editor to 4 spaces.